docs/getting-started.md

```diff
@@ -0,0 +1,181 @@
+﻿# Kubevirtci
+
+The purpose of Kubevirtci is to create pre-provisioned Kubernetes/OpenShift clusters as container images,\
+allowing people to easily run a of a k8s / OpenShift cluster.
+
+The target audience should be newcoming developers of kubevirtci.
```

Comment:
I think that

The target audience *is* people wanting to create a new kubevirtci provider or freshen and existing one.